### PR TITLE
Very rough read-only PersonalInfo 2.0

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
This begins to lay the groundwork for the new Personal and Contact Info section for Profile 2.0.

It is currently read only and there is a lot of work still to be done. But this at least gets some stuff in place.

## Testing done
Just a little local testing + existing unit tests

## Screenshots
<details>
  <summary>Desktop</summary>
  
image

</details>

<details>
  <summary>Mobile</summary>
  
image

</details>

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by sanlouise — APPROVED** [review]

File: src/applications/personalization/profile-2/components/personal-information/AddressesTable.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  const content = (
+    <div className={classes.wrapper}>
+      {contactInfo}
+      <button className={classes.editButton}>Edit</button>
+    </div>
+  );
+  return content;
+};
+
+const AddressesTable = ({ className, homeAddress, mailingAddress }) => (
```
[the remark this answers is not in the document]
Comment (erikphansen) [reply src/applications/personalization/profile-2/components/personal-information/AddressesTable.jsx:50]: I'm not sure I follow... We'd just have to move most of this markup up a level into the `ContactInformationSection` component.

File: src/applications/personalization/profile-2/components/personal-information/GenderAndDOBSection.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+const renderGender = gender => {
+  let content = notSetText;
+  if (gender === 'M') content = 'Male';
+  else if (gender === 'F') content = 'Female';
+  return content;
+};
+
+const renderDOB = dob => {
+  let content = notSetText;
```
[the remark this answers is not in the document]
Comment (erikphansen) [reply src/applications/personalization/profile-2/components/personal-information/GenderAndDOBSection.jsx:19]: good idea

File: src/applications/personalization/profile-2/components/personal-information/PersonalInformation.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from 'react';
+
+import { focusElement } from 'platform/utilities/ui';
+
+import PersonalInformationContent from './PersonalInformationContent';
+
+const PersonalInformation = () => {
+  useEffect(() => {
+    focusElement('[data-focus-target]');
+  }, []);
```
Comment (ncksllvn) [inline src/applications/personalization/profile-2/components/personal-information/PersonalInformation.jsx:10]: I used `useEffect` like this once while messing around with Create React App. I got a great big linter error about the `[]` passed in. This is a totally legit way to do the same thing as `componentDidMount` though, right? For my own learning 🙂

Comment (erikphansen) [reply src/applications/personalization/profile-2/components/personal-information/PersonalInformation.jsx:10]: yeah, this is how you do `componentDidMount`, more or less

**Pull request merged**